lesson_22/script.js

```diff
@@ -0,0 +1,39 @@
+
+let number1 = prompt('Введіть перше число для додавання, віднімання, множення та ділення');
+let number2 = prompt('Введіть друге число для додавання, віднімання, множення та ділення');
+
+if (!number1 || !number2 || (isNaN(number1) || isNaN(number2))) {
+    alert('Обидва рядки повинні бути заповненими та введені значення повинні бути числами.');
+} else {
+
+    let result;
+
+    if (+number1 < +number2) {
```

Comment:
it's better to override values at the top:

```
number1 = +number1;
number2 = +number2;
```

to not add `+` every time when you use these variables